docker-service/templates/docker-compose-service

```diff
@@ -24,10 +32,20 @@ start()
 
 stop()
 {
-    /usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml stop
+    echo "stopping /etc/docker-compose/{{service_name}}/docker-compose.yml ..."
+    /usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml down
     /usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml kill
 }
 
+status()
+{
+    STATUS=`/usr/local/bin/docker-compose -p {{service_name}} -f /etc/docker-compose/{{service_name}}/docker-compose.yml ps`
+    if ! echo $STATUS | grep Up; then
+        echo "service /etc/docker-compose/{{service_name}}/docker-compose.yml not running..."
```

Comment:
may be better to just print the output of the docker-compose ps as per the previous comment on exited containers